include/mc_tasks/ImpedanceTask.h

```diff
@@ -196,6 +196,24 @@ struct MC_TASKS_DLLAPI ImpedanceTask : SurfaceTransformTask
     lowPass_.cutoffPeriod(cutoffPeriod);
   }
 
+  /*! \brief Get whether hold mode is enabled. */
+  bool hold() const noexcept
+  {
+    return hold_;
+  }
+
+  /*! \brief Set hold mode.
+   *
+   *  In hold mode, the compliance modification (deltaCompPoseW_) is automatically updated so that the final target pose
+   * sent to the QP (i.e. compliancePose()) remains constant even if the user-specified target pose changes. This is
+   * useful when the user wants to change the target pose without moving the object held by the robot (e.g., when there
+   * is an error in the object pose and it is corrected based on vision sensor measurements).
+   */
+  void setHold(bool hold)
```

Comment:
`inline` and `noexcept`, and `hold(bool)` rather than `setHold(bool)`

Reply:
Fixed in https://github.com/jrl-umi3218/mc_rtc/pull/143/commits/51d1eb078d2d27fc83085dd1e48545344e8face7